templates/taxi/car_delete.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Car</h1>
+  <p>
+    Are u sure, that u want to delete car {{ car }}?
+  </p>
+  <p><i>All info about this car will be irretrievably deleted</i></p>
+  <form action="" method="post">  
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-secondary">
+    <a href="{% url 'taxi:car-list' %}" class="btn btn-danger">Cancel</a>
```

Comment:
The 'Yes, delete' button should have the 'btn-danger' class, and the 'Cancel' button should have the 'btn-secondary' class, as per common UI conventions. This helps users easily identify the destructive action.